src/core/formatters.ts

```diff
@@ -199,6 +208,7 @@ export module Formatters {
    * @returns {Formatter} A formatter for time/date values.
    */
   export function relativeDate(baseValue: number = 0, increment: number = MILLISECONDS_IN_ONE_DAY, label: string = "") {
+    Plottable.Utils.Window.deprecated("relativeDate()", "1.2", "Not safe for use with time zones.");
```

Comment:
Is there some form of satisfiable replacement that we can list in the message parameter?  One reason the message parameter exists is so we can put messages on what the user should use instead.

The lucky thing in this particular instance is that since Formatters are so lightweight, we probably can tell users to just "make their own".  Not sure if this is the right route to go.